.github/workflows/on-push.yml

```diff
@@ -5,14 +5,23 @@ env:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        otp: ['23.3.4.11']
+        elixir: ['1.9.4', '1.13.2']
     steps:
       - uses: actions/checkout@v2
       - uses: erlef/setup-elixir@v1
         with:
-          otp-version: '23.0.4'
-          elixir-version: '1.10.4'
+          otp-version: ${{matrix.otp}}
+          elixir-version: ${{matrix.elixir}}
       - uses: rrainn/dynamodb-action@v2.0.0
+      - run: mix deps.update --all
+        if: matrix.elixir == '1.13.2'
       - run: mix deps.get
       - run: mix compile
       - run: mix format --check-formatted
+        if: matrix.elixir == '1.13.2'
+      - run: mix dialyzer
+        if: matrix.elixir == '1.13.2'
```

Comment:
I don't know the yml syntax for Github particularly well, but would there be a way that both of these `run` tasks could be wrapped under one condition? Like

```
- run: mix format --check formatted && mix dialyzer
  if: matrix.elixir == '1.13.2'
```

I don't know if that's right or not, just meant to be illustrative. If not, no big deal, this is cool.

Reply:
Thanks！I fixed it.